plugins/chat/assets/javascripts/discourse/components/chat-message-thread-indicator.hbs

```diff
@@ -8,50 +8,26 @@
   role="button"
   title={{i18n "chat.threads.open"}}
 >
-  {{#unless this.chatStateManager.isDrawerActive}}
-    <div class="chat-message-thread-indicator__last-reply-avatar">
-      <ChatUserAvatar
-        @user={{@message.thread.preview.lastReplyUser}}
-        @avatarSize="small"
-      />
-    </div>
-  {{/unless}}
-  <div class="chat-message-thread-indicator__body">
-    <div class="chat-message-thread-indicator__last-reply-metadata">
-      <div class="chat-message-thread-indicator__last-reply-user">
-        {{#if (or this.site.mobileView this.chatStateManager.isDrawerActive)}}
-          <span
-            class="chat-message-thread-indicator__last-reply-avatar -mobile"
-          >
-            <ChatUserAvatar
-              @user={{@message.thread.preview.lastReplyUser}}
-              @avatarSize="tiny"
-            />
-          </span>
-        {{/if}}
-        <span class="chat-message-thread-indicator__last-reply-username">
-          {{@message.thread.preview.lastReplyUser.username}}
-        </span>
-      </div>
-      <span class="chat-message-thread-indicator__last-reply-container">
-        {{format-date
-          @message.thread.preview.lastReplyCreatedAt
-          leaveAgo="true"
-          prefix=(i18n "chat.thread.last_reply")
-        }}
-      </span>
-      <span class="separator">|</span>
-      <span class="chat-message-thread-indicator__replies-count">
-        {{i18n "chat.thread.replies" count=@message.thread.preview.replyCount}}
-      </span>
 
-    </div>
-    <div class="chat-message-thread-indicator__last-reply-excerpt">
-      {{replace-emoji (html-safe @message.thread.preview.lastReplyExcerpt)}}
-    </div>
+  <div class="chat-message-thread-indicator__last-reply-avatar">
+    <ChatUserAvatar
+      @user={{@message.thread.preview.lastReplyUser}}
+      @avatarSize="small"
+    />
   </div>
 
-  <div class="chat-message-thread-indicator__participants">
-    <Chat::Thread::Participants @thread={{@message.thread}} />
+  <div class="chat-message-thread-indicator__last-reply-info">
+    <span class="chat-message-thread-indicator__last-reply-username">
+      {{@message.thread.preview.lastReplyUser.username}}
+    </span>
+    <span class="chat-message-thread-indicator__last-reply-timestamp">
+      {{format-date @message.thread.preview.lastReplyCreatedAt leaveAgo="true"}}
+    </span>
+  </div>
+  {{! <div class="chat-message-thread-indicator__participants"> }}
```

Comment:
did you mean to remove this?

Reply:
Remove what?

Reply:
The comment? `{{! <div class="chat-message-thread-indicator__participants"> }}` ?